demo/locales/en.json

```diff
@@ -169,6 +169,7 @@
   "DEMO_SURROUND_SEARCH": "Filters for assets with at least one surround sound audio track.",
   "DEMO_SWITCH_INTERVAL": "Switch Interval",
   "DEMO_TEXT_LANGUAGE": "Preferred Text Language",
+  "DEMO_TEXT_LANGUAGE_FORCED": "Preferred Text Language Forced",
```

Comment:
Can we please update this message as well?  "Prefer Forced Subs"

I would also prefer that we update the constants (DEMO_PREFER_FORCED_SUBS)

Reply:
Done!